gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -65,15 +65,38 @@
           </div>
           <div class="row">
             <div class="col-md-12">
-              <div class="form-group">
-                <label translate for="folders">Share Folders With Device</label>
+              <div class="form-group" ng-if="currentSharing.shared.length">
+                <label translate for="folders">Currently Shared Folders for Device</label>
```

Comment:
I'd prefer

```suggestion
                <label translate for="folders">Shared Folders</label>
```

as in the untrusted. There's no need to mention the context of a device, when in the device edit modal.

Reply:
Right, that's probably too much.  I was trying to align it with the edit folder modal, which has a very similar heading.  Will change.